examples/gemini/javascript/langchain_quickstart_node/main.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { readFile } from 'node:fs/promises';
+import { ChatGoogleGenerativeAI } from '@langchain/google-genai';
+import { HumanMessage } from '@langchain/core/messages';
+
+/**
+ * Creates a Gemini Pro text-only chat model, invokes the model with a single
+ * input, and logs the result.
+ */
+async function invokeGeminiPro() {
+  const model = new ChatGoogleGenerativeAI({
+    modelName: 'gemini-pro',
+    maxOutputTokens: 1024,
+  });
+
+  const result = await model.invoke([
+    [
+      'human',
+      'What is the meaning of life?',
+    ],
+  ]);
+
+  console.log(result);
+}
+
+/**
+ * Creates a Gemini Pro Vision multimodal chat model, invokes the model with an
```

Comment:
This is a little confusing (not your comment, just the structure of things) - we don't actually expose a multimodal chat model, but I think LC is wrapping up this "Chat" service using the straight text-completion service. The class is called "ChatGoogleGenerativeAI" so I think it's fine, just potentially confusing.